app/services/idv/steps/verify_step.rb

```diff
@@ -1,8 +1,70 @@
 module Idv
   module Steps
     class VerifyStep < VerifyBaseStep
+      State = Struct.new(:status, :pii, :result)
+
       def call
-        perform_resolution_and_check_ssn
+        case async_state.status
+        when :none
+          enqueue_job
+        when :in_progress
+          nil
+        when :timed_out
+          enqueue_job
+        when :done
+          nil
+        end
+      end
+
+      def after_call(pii, idv_result)
+        # binding.pry
+
+        add_proofing_costs(idv_result)
+        response = idv_result_to_form_response(idv_result)
+        response = check_ssn(pii) if response.success?
+        summarize_result_and_throttle_failures(response)
+      end
+
+      def async?
+        true
+      end
+
+      # @return [State]
+      def async_state
+        dcs_uuid = flow_session[:idv_verify_step_document_capture_session_uuid]
+        dcs = DocumentCaptureSession.find_by(uuid: dcs_uuid)
+        return State.new(:none, nil, nil) if dcs_uuid.nil?
+        return State.new(:timed_out, nil, nil) if dcs.nil?
```

Comment:
Another idea from chat, let's make separate initialization methods for this struct? So that it's harder to initialize incorrectly? Also use `keyword_init` might make it easier:

```
State = Struct.new(:status, :pii, :result, keyword_init: true) do
  def self.none
    new(status: :none)
  end

  def self.timed_out
    new(status: :timed_out)
  end
 
  def self.done(pii:, result:)
    new(status: :done, pii: pii, result: result)
  end

  private_class_method :new
end
```

Then this can be:

```suggestion
        return State.none if dcs_uuid.nil?
        return State.timed_out if dcs.nil?
```